### PR TITLE
<Carousel/> - Set 'clickNext' and 'clickPrevious' as public to driver's consumers [Testkit].

### PR DESCRIPTION
**Pull request by netanel89**
### 🔦 Summary

`Carousel` driver is not supporting for arrow click simulation in tests.
That is because those event triggers are set as private ( in the private drivers ).
I just moved those triggers to the public drivers.

### ✅ Checklist

- 📚 Change is documented
  - [X] Story
  - [X] API description

### CODE REVIEW

File: src/Carousel/Carousel.uni.driver.js [hunk 1]
```diff
@@ -28,5 +28,17 @@ export const carouselUniDriverFactory = base => {
...
+     * Simulate a click on the previous item button
+     * @return {Promise<void>}
+     */
+    clickPrevious: () => base.$('[data-hook="prev-button"]').click(),
+
+    /**
+     * Simulate a click on the next item button
+     * @return {Promise<void>}
+     */
+    clickNext: () => base.$('[data-hook="next-button"]').click(),
```
Comment (EgoziE) [inline src/Carousel/Carousel.uni.driver.js:42]: Please update `src/Carousel/Carousel.uni.driver.d.ts` accordingly

File: src/Carousel/Carousel.driver.js [hunk 2]
```diff
@@ -18,5 +20,13 @@ export const carouselDriverFactory = ({ element }) => {
...
       ].map(img => img.src);
     },
+    clickPrevious: () => {
+      const prevButton = element.querySelector('[data-hook="prev-button"]');
+      ReactTestUtils.Simulate.click(prevButton);
+    },
+    clickNext: () => {
+      const nextButton = element.querySelector('[data-hook="next-button"]');
+      ReactTestUtils.Simulate.click(nextButton);
+    },
```
Comment (EgoziE) [inline src/Carousel/Carousel.driver.js:30]: Please update `src/Carousel/Carousel.driver.d.ts` accordingly

**Review by EgoziE — CHANGES_REQUESTED** [review]
Thanks for this PR!

Please take the time to read our guide [How to create a PR for wix-style-react](https://github.com/wix/wix-style-react/blob/master/docs/contribution/CREATE_PR.md), you must sign your commits in order for CI to run.

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6303

**Comment by netanel89** [comment]
@EgoziE alright ! seems to be just fine now ..

**Pull request merged**